streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -1262,6 +1266,7 @@ public void run() {
                     }
                     incrementIteration();
                 }
+                producer.flush();
```

Comment:
The producer is create using try-with-resource clause thus, the producer should be flushed on the implicit `close()` anyway? Thus, I don't think explicit flushing would help?

Reply:
Makes sense, will fix.